README.md

```diff
@@ -86,6 +86,22 @@ virtualenv -p python3 env
 pip install -e microraiden
 ```
 
+* if necessary to complete the build, install latest OpenSSL on MacOS
```

Comment:
Please add this after `Quick Start`:
### MacOS build issues with OpenSSL
<your suggestions>

Reply:
I think this is what you're looking for, let me know if I can provide further improvement.